### DISCUSSION TITLE
how to submit tests to the platform?

### DISCUSSION THREAD

**Discussion by elio4sakk** [SwissClinicalTrialOrganisation/validation_platform#82] [General]
I want to upload tests to the platform. And for this: 1) I navigated to the test folder, 2) I clicked on "Add file", and 3) I clicked on "Upload files". I was expecting to be able to drag and drop my folder with the tests; however, I got the message: "Uploads are disabled. File uploads require push access to this repository." Can anyone please help me solve this problem?

**Comment by aghaynes** [comment]
Does it let you if you fork the repository to your account first?

**Comment by aghaynes** [comment]
elio4sakk has since successfully submitted tests to the platform, so I'm closing this
